### DISCUSSION TITLE
Looking for Wix Installer with User Interface

### DISCUSSION THREAD

**Discussion by TimoBSchott** [wixtoolset/issues#8137] [Questions]
We've been using Wix Installer 3.1 for some time now. While it works, we often find ourselves spending a lot of time training new team members or creating entirely new scripts because there is no IntelliSense or graphical user interface to assist with scripting. When it comes to custom actions, the sequence of installer actions, and conditions, there are always uncertainties. Neither the sequence nor the keywords can be intuitively chosen. Additionally, copying previously set custom variables can only be done manually.

We would like to know what tools you use or if there is a graphical Wix interface available for editing installer scripts. Something like what's available in InnoSetup, where you have a modern, visually-oriented interface with color-coding and ease of use.

![project-properties](https://github.com/wixtoolset/issues/assets/94844207/41ed6ba8-ec99-4dcb-b2ac-a66c68a0f9e4)

**Comment by chrpai** [comment]
I have a FOSS project on GitHub called IsWiX. It gives you project templates (scaffolding) and designers for the main things like Files and Folders, Services and ShortCuts.  (like 95%-99% of any given install.) The rest is still done by hand for 2 reasons:  1) No one ever joined in to help create additional designers and 2) I didn't want to help with things like Custom Actions because they are an advanced topic which can be abused and the actual wiring up part isn't the hard part anyways.

Google IsWiX Tutorials and find my email address if you'd like a personal demonstration.
